README.md

```diff
@@ -86,7 +86,11 @@ chmod +x build-s3-dist.sh
 
 ## Deploy
 
-- Deploy the distributable to the Amazon S3 bucket in your account. Make sure you are uploading the files in `deployment/global-s3-assets` and `deployment/regional-s3-assets` to `$BUCKET_NAME/$SOLUTION_NAME/$VERSION`.
+- If it does not already exist, create the Amazon S3 bucket called $BUCKET_NAME in your account which will be used to store the distributable files.
+- Deploy the distributable files to the Amazon S3 bucket in your account. Make sure you are uploading the files in `deployment/global-s3-assets` and `deployment/regional-s3-assets` to `$BUCKET_NAME/$SOLUTION_NAME/$VERSION`.
+- An example looks like the following
+  - aws s3 cp  global-s3-assets/$SOLUTION_NAME.template s3://$BUCKET_NAME/$SOLUTION_NAME/$VERSION/
```

Comment:
__nit__: the command could be enclosed in backticks, like `aws s3 cp global-s3-assets/$SOLUTION_NAME.template s3://$BUCKET_NAME/$SOLUTION_NAME/$VERSION/`. The same for the line below.

Reply:
PR revised, tks for the comment